HPXMLtoOpenStudio/resources/output.rb

```diff
@@ -986,200 +986,103 @@ def self.write_debug_files(runner, model, weather, debug, output_dir)
-        if service_feeder.type == HPXML::ElectricPanelLoadTypeHeating
-          service_feeder.components.each do |component|
-            component.branch_circuits.each do |branch_circuit|
-              htg += branch_circuit.occupied_spaces * unit_multiplier
-            end
-          end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypeCooling
-          service_feeder.components.each do |component|
+        service_feeder.components.each do |component|
+          if service_feeder.type == HPXML::ElectricPanelLoadTypeCooling
             # So we don't double-count, e.g., heat pump breaker spaces since it is shared across two service feeder types
             next unless component.service_feeders.select { |sf| sf.type == HPXML::ElectricPanelLoadTypeHeating }.empty?
-
-            component.branch_circuits.each do |branch_circuit|
-              clg += branch_circuit.occupied_spaces * unit_multiplier
-            end
           end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypeWaterHeater
-          service_feeder.components.each do |component|
-            component.branch_circuits.each do |branch_circuit|
-              hw += branch_circuit.occupied_spaces * unit_multiplier
-            end
-          end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypeClothesDryer
-          service_feeder.components.each do |component|
-            component.branch_circuits.each do |branch_circuit|
-              cd += branch_circuit.occupied_spaces * unit_multiplier
-            end
-          end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypeDishwasher
-          service_feeder.components.each do |component|
-            component.branch_circuits.each do |branch_circuit|
-              dw += branch_circuit.occupied_spaces * unit_multiplier
-            end
-          end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypeRangeOven
-          service_feeder.components.each do |component|
-            component.branch_circuits.each do |branch_circuit|
-              ov += branch_circuit.occupied_spaces * unit_multiplier
-            end
-          end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypeMechVent
-          service_feeder.components.each do |component|
-            component.branch_circuits.each do |branch_circuit|
-              vf += branch_circuit.occupied_spaces * unit_multiplier
-            end
-          end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypePermanentSpaHeater
-          service_feeder.components.each do |component|
-            component.heater_branch_circuits.each do |branch_circuit|
-              sh += branch_circuit.occupied_spaces * unit_multiplier
-            end
-          end
-        elsif service_feeder.type == HPXML::ElectricPanelLoadTypePermanentSpaPump
-          service_feeder.components.each do |component|
-            component.pump_branch_circuits.each do |branch_circuit|
```

Comment:
Buried, but this is `pump_branch_circuits` and not `branch_circuits`.

@shorowit